README.md

```diff
@@ -14,6 +14,14 @@ See [UPDATES.md](UPDATES.md).
 
 ⚠️ **7/18: We're aware of people encountering a number of download issues today. Anyone still encountering issues should remove all local files, re-clone the repository, and [request a new download link](https://ai.meta.com/resources/models-and-libraries/llama-downloads/). It's critical to do all of these in case you have local corrupt files. When you receive the email, copy *only* the link text - it should begin with https://download.llamameta.net and not with https://l.facebook.com, which will give errors.**
 
+
+### Download
+Once your request is approved, you will receive links to download the tokenizer and model files.
+
+Edit the `download.sh` script with the signed url provided in the email to download the model weights and tokenizer.
+
+The download process can also be automated by setting the respective environmental variables (`PRESIGNED_URL` and `TARGET_FOLDER`).
+
```

Comment:
Most of this feel redundant to me. I suggest instead to just add a example below:

"Then to run the script: `./download.sh`. There are optional ENV variables that can be used for automation.

"Then run the download.sh script, passing the URL provided when prompted to start the download (or using the )".

```suggestion
```